packages/angular/src/directives/navigation/ion-router-outlet.ts

```diff
@@ -415,3 +433,93 @@ class OutletInjector implements Injector {
     return this.parent.get(token, notFoundValue);
   }
 }
+
+export const INPUT_BINDER = new InjectionToken<RoutedComponentInputBinder>('');
```

Comment:
Can we add TODOs/a spike in our backlog so we know to remove this once Angular <16 is eventually dropped?